deploy/cr.yaml

```diff
@@ -185,6 +185,7 @@ spec:
   orchestrator:
     image: perconalab/percona-server-mysql-operator:main-orchestrator
     imagePullPolicy: Always
+    serviceAccountName: percona-server-mysql-operator-orchestrator
```

Comment:
Can we use 'percona-server-mysql-operator-orchestrator' by default and comment this line in CR?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/227/commits/4971e28e59c8e5076f25c79215ca832bb9e22fa7